.github/workflows/default.yml

```
@@ -0,0 +1,144 @@
+          - component: nginx
+            debug: false
+          - component: trivy-adapter
+            debug: false
+          - component: cmd/exporter
+            debug: false
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set component vars
+        id: comp
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:6:44: Double quote to prevent globbing and word splitting

(shellcheck)